util/getImageMimeType.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Naive mime-type function
+ */
+module.exports = (extension) => {
+  extension = extension.replace(".", "");
+  if (extension === "jpg") {
+    return "image/jpeg";
+  }
+  if (extension === "svg") {
+    return "image/svg+xml";
+  }
+  return `image/${extension}`;
+};
```

Comment:
This file has been renamed to camel-case, but note that `deflate-image.js` still has a dashed filename.